contrib/devtools/check-doc.py

```diff
@@ -33,12 +33,12 @@ def main():
   args_need_doc = args_used.difference(args_docd)
```

Comment:
One line above this:

You are switching this file to python3, so `check_output` will return byte-strings instead of text strings. Thus, you can not apply a regex on them.

Reply:
Should be fixable if you switch the patterns to bytestrings `b'pattern'`